app/src/main/java/org/gdg/frisbee/android/activity/GdeActivity.java

```diff
@@ -51,8 +52,8 @@ public void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.activity_gde);
 
-        getSupportActionBar().setLogo(R.drawable.ic_gde_logo_wide);
-        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        Toolbar toolbar = getActionBarToolbar();
+        toolbar.setTitle(R.string.gde);
```

Comment:
I know, I'll have to fix that later, we need to merge this ASAP so that you can continue working.